operations/before.go

```diff
@@ -194,6 +194,15 @@ func requireAtLeastOneBool(flags ...string) cli.BeforeFunc {
 	}
 }
 
+func requireAgentFlags(beforeFuncs ...func(c *cli.Context) error) []func(c *cli.Context) error {
+	return append(beforeFuncs,
+		requireStringFlag(apiServerURLFlagName),
+		requireStringFlag(hostIDFlagName),
+		requireStringFlag(hostSecretFlagName),
+		requireStringFlag(workingDirectoryFlagName),
+	)
+}
```

Comment:
I'd just do "mergeBeforeFuncs()" to avoid needed to do awkward things with variadic functions.